client/src/components/navigations/MainStackNavigator/ProfileModal.tsx

```diff
@@ -408,29 +432,36 @@ const ModalContent: FC<ModalContentProps> = ({modalState, hideModal}) => {
 
         {!modalState?.hideButtons ? (
           <StyledViewBtns>
-            {deleteFriendInFlight ? (
-              <LoadingIndicator size="small" />
-            ) : (
-              <TouchableOpacity
-                testID="touch-add-friend"
-                activeOpacity={0.5}
-                onPress={isFriend ? deleteFriend : addFriend}
-                style={styles.viewBtn}>
-                <View style={styles.viewBtn}>
-                  <StyledText testID="text-add-title">
-                    {isFriend
-                      ? getString('DELETE_FRIEND')
-                      : getString('ADD_FRIEND')}
-                  </StyledText>
-                </View>
-              </TouchableOpacity>
+            {!modalState.isMyself && (
+              <>
+                {deleteFriendInFlight ? (
+                  <LoadingIndicator size="small" />
+                ) : (
+                  <TouchableOpacity
+                    testID="touch-add-friend"
+                    activeOpacity={0.5}
+                    onPress={isFriend ? deleteFriend : addFriend}
+                    style={styles.viewBtn}>
+                    <View style={styles.viewBtn}>
+                      <StyledText testID="text-add-title">
```

Comment:
Now I know that `<View>` component which wraps `<StyledText>` is unnecessary at all. So I remove all those `<View>` components instead of separating styles.

Reply:
Great